Client/src/Pages/Register/index.jsx

```diff
@@ -150,7 +150,7 @@ const Register = () => {
         <div className="register-form-checks">
           <Check text="Must be at least 8 characters" />
           <div className="register-form-v-spacing-small"></div>
-          <Check text="Must contain one special character" />
+          <Check text="Must contain one special character, one number and one uppercase" />
```

Comment:
I would change this to:

"Must contain one special character, one number, and one uppercase letter"

I'm a stickler for the [oxford comma](https://en.wikipedia.org/wiki/Serial_comma) and the last "letter" is needed